src/components/NavigationBar/style.css

```diff
@@ -16,7 +16,7 @@
   height: 32px;
   color: #2C3E4F;
   font-size: 14px;
-  line-height: 20px;
+  line-height: 17px;
```

Comment:
Is this right? `17px` does not feel like a value we would use in the design framework

Reply:
Yes, @danielspagnolo had the impression that the arrow wasn't properly aligned with the text so we tested it on the browser and 17px was the value that was making it look good. Right @danielspagnolo?

Reply:
@hjheath yes you're right that the 17px doesn't fit into our framework, but we also have to be slightly flexible in situations like this, where optically the "back" copy doesn't look right to the human eye when sitting next to the arrow icon